src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -137,7 +137,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $validAssetDirs = [];
         /** @var BundleInterface $bundle */
         foreach ($kernel->getBundles() as $bundle) {
-            if (!is_dir($originDir = $bundle->getPath().'/Resources/public')) {
+            if (!method_exists($bundle, 'getPublicPath')) {
+                @trigger_error('Not definind getPublicPath() method is deprecated since Symfony 4.4 and will not be supported in 5.0.', E_USER_DEPRECATED);
+                $publicPath = '/Resources/public';
+            } else {
+                $publicPath = $bundle->getPublicPath();
+            }
+            if (!is_dir($originDir = $bundle->getPath().$publicPath)) {
```

Comment:
As the path is relative, you should add `DIRECTORY_PATH` between getPath() and $publicPath.